packages/web-components/@groupby/elements-base/test/unit/base.test.ts

```diff
@@ -1,5 +1,86 @@
-import { expect, stub } from '../utils';
+import { expect, stub, spy } from '../utils';
```

Comment:
Do we have a consistent order for these? If we do, it'll compress better.

Reply:
I made it consistent with the others. This was autoimported and put there and didn't notice. My b.